### PR TITLE
feat(files): Refresh grid view layout

### LINKED ISSUE #45438: Make the grid view more usable by handling longer filenames better
Improve the overall design of the grid view.
@nextcloud/designers for mockups

* Main issue is that file names beyond ~10 characters can't be read. 
* If accessibility can be fixed/improved easily, do that too, but only as low priority.

### PR DESCRIPTION
**Pull request by artonge**
- Fix https://github.com/nextcloud/server/issues/45438

## Mockups

341374888-2f82d769-475b-4717-9580-d93a9a29cac2

## Current state

Before | Screenshots
-- | --
Overview | Screenshot from 2024-07-04 16-43-38
Focus states | Screenshot from 2024-07-04 16-43-59 Screenshot from 2024-07-04 16-44-01 Screenshot from 2024-07-04 16-44-03

### CODE REVIEW

File: apps/files/src/components/FileEntryMixin.ts [hunk 1]
```diff
@@ -148,8 +156,22 @@ export default defineComponent({
 			},
 		},
 
-		isRenaming() {
```
Comment (skjnldsv) [inline apps/files/src/components/FileEntryMixin.ts:151]: Where did this variable go? :eyes:

Comment (skjnldsv) [reply apps/files/src/components/FileEntryMixin.ts:151]: Ah funny, it was duplicated! Let's go!! :ship:

Comment (artonge) [reply apps/files/src/components/FileEntryMixin.ts:151]: Was somehow a duplicate. The other one is a bit up

**Review by skjnldsv — APPROVED** [review]

**Review by nfebe — APPROVED** [review]

**Pull request merged**